test/bindings/http/unmarshaller_0_3_tests.js

```diff
@@ -108,10 +108,11 @@ describe("HTTP Transport Binding Unmarshaller for CloudEvents v0.3", () => {
       const un = new Unmarshaller();
 
       // act and assert
-      un.unmarshall(payload, headers)
-        .then(() => { throw new Error("failed"); })
-        .catch((err) =>
-          expect(err.message).to.equal("invalid payload"));
+      try {
+        await un.unmarshall(payload, headers);
```

Comment:
@helio-frota you should fail the test after this line. The expectation is that the unmarshalling should throw if the incoming message does not conform to spec. As the test is now, that isn't happening. We never enter the catch block below. You can change that `.to.equal()` call to be `.to.equal('tacos')` and the test still passes.

Reply:
OOps you right! I was only focused on solve the unhandled promise exception.
Thanks for the review, commit updated.

Reply:
It still not good btw..
I'm trying to understand what is happening...